src/test/scala/com/cognite/sdk/scala/v1/SessionsTest.scala

```diff
@@ -369,13 +370,13 @@ class SessionsTest extends SdkTestSpec with ReadBehaviours {
         )
       )
 
-    val client = new GenericClient[Id](
+    val client = new GenericClient[IO](
       applicationName = "CogniteScalaSDK-OAuth-Test",
       projectName = "session-testing",
       auth = BearerTokenAuth("bearer Token")
     )(implicitly, implicitly, responseForSessionRevoke)
 
-    val responseDelete = client.sessions.revoke(Items(expectedIds))
+    val responseDelete = client.sessions.revoke(Items(expectedIds)).unsafeRunSync()
```

Comment:
will need to check whether we can avoid IO here, couple options
- Id[_] -> Either[Throwable, _]
- pull IdMonad implementation from sttp that throws jvm exceptions

Reply:
yep, either is an option